server/src/mongo/models/Beatmap.ts

```diff
@@ -103,6 +104,7 @@ const schema: Schema = new Schema({
       required: true,
       type: String,
     },
+    songDuration: { type: Number, default: 0 },
```

Comment:
Some form of validation on this (ie cannot be negative) would be nice

Reply:
Good recommendation.  Some validation is always nice